src/platform/CMakeLists.txt

```diff
@@ -53,3 +55,7 @@ add_subdirectory(client)
 add_subdirectory(console)
 add_subdirectory(logger)
 add_subdirectory(update)
+
+if (JOURNALD_FOUND AND NOT MULTIPASS_ENABLE_SYSLOG)
+  add_definitions(-DJOURNALD_ENABLED)
```

Comment:
I'd also prefix this definition with `MULTIPASS_`, so `MULTIPASS_JOURNALD_ENABLED` is coherent with most of the definitions added here.